### ISSUE TITLE
review c21, Abrams Creek

### ISSUE THREAD

**Issue by jrosen48** [jrosen48/little-kids-big-trips#4]
review c21, Abrams Creek

It is in the file: [`05-02-abrams.Rmd`](`05-02-abrams.Rmd`)

That file is rendered here: https://littlekidsbigadventures.com/abrams-creek.html

General areas in need of feedback:

- accessibility to our audience
- coherence and clarity of "voice"
- inclusion of "parent voice": making sure that we are communicating to someone who may be the parent of a young kiddo new to the outdoors
- should any images be added?
- whether there are any other ideas/notes/elements to be added

Specific area(s) in need of feedback:

- who farmed the area that is now traversed by the 'little bottoms trail'? This is near what is presently backcountry site #17. many guides mention that this area was farmed prior to the formation of the national park. The little brown book that we have may have details on this. Ken Wise's book may have details on this. 

How to add comments:

- add a comment below!

**Comment by krosenberg3** [comment]
anything

**Comment by krosenberg3** [comment]
General areas in need of feedback:
accessibility to our audience
Yes, I do think it’s accessible. One recommendation is to keep the addition of nature/history sections short and to the point. 
coherence and clarity of "voice"
It’s definitely all very clear. Sometimes, while reading, it feels like you veer off to researcher mode every-so-slightly. Maybe write this as though Akram is your ideal audience. :-)
I recommend taking out parenthesis…basically always. Either don’t include that information, or just set it off with commas.
inclusion of "parent voice": making sure that we are communicating to someone who may be the parent of a young kiddo new to the outdoors
After reading through this chapter, it occurs to me that one way to enhance this is to give short, personal insights to how our own kid does on the trails. For instance, what about mentioning things Jonah likes to do at the creek crossings?
should any images be added? 
Not sure about adding images, but I really don’t like the one of my feet up the tree holding my phone out to take a pic. :-)
whether there are any other ideas/notes/elements to be added
Specific area(s) in need of feedback:
who farmed the area that is now traversed by the 'little bottoms trail'? This is near what is presently backcountry site #17. many guides mention that this area was farmed prior to the formation of the national park. The little brown book that we have may have details on this. Ken Wise's book may have details on this.
I didn’t find much, and certainly nothing substantive, but: 
https://www.hikingproject.com/trail/7008278/little-bottoms-trail
Not related to bottomlands, but: “Before the first white settler arrived on the scene, Cades Cove had been a part of the Cherokee Indians' domain. Abrams Creek and Abrams Falls are features named for a prominent Cherokee chief named "Old Abram" who at one time lived in a village at Chilhowee on the Little Tennessee River. According to tradition, Old Abram's wife was named Kate, and Cades (Kate's) Cove was named after her.” - http://www.yoursmokies.com/cades-cove-history.html
http://smokyscout.blogspot.com/2010/08/so-good-to-be-back-again.html?m=1
How to add comments:
add a comment below!
________________________________________________________
21.1.2 Highlights: 
You might want to add that a way to extend the hike or to level up the challenge level for kids/families who are ready is to take the trail that goes up above the campground beyond the creeks
21.1.3:
If you add the above bullet point in highlights, you could perhaps change the rating to 2-4
One thought on the rating: You might
 
21.2:
Along with trail overview information, this would be a good spot to give parents a head’s up of anything out of the ordinary. For instance, if I were a parent, new to hiking, I would want to know about all the creek crossings and to plan accordingly (waterproof shoes, hiking sandals that are made to get wet, recommend to have dry clothes/shoes in your backpack or in the car)

21.4:
Hemlock information is a bit long. I think these informational sections should be packed with information, but be as concise as possible. Example:
Do you see the tall pine trees here at the campground? Many of them are Eastern Hemlocks, one of the tallest trees in the Eastern United States. Hemlocks can be identified by their flat needles that exhibit a flat arrangement on the trees’ branches. They can take 250 to 300 years to reach maturity and can live for an astonishing 800 years or more. They can grow to far taller than 150 feet. The tallest Eastern Hemlock in the world is in the Smokies and [is around 170 feet tall](https://www.monumentaltrees.com/en/trees/tsugacanadensis/records/)!
You may notice that some of the trees have what appears to be a painted marking on their base. This is a sign the trees have been infected by an invasive insect, the Hemlock Wooly Adelgid. Thankfully, there is an effective, relatively easy way to treat the trees. A chemical similar to anti-tick sprays for pets is dissolved in water and poured around the base of the tree. The park service has treated more than 200,000 trees in this way. Learn more about the Hemlock Wooly Adelgid and its prevention in [Service](https://littlekidsbigadventures.com/abrams-creek.html#ref-nps_hwa) ([n.d.](https://littlekidsbigadventures.com/abrams-creek.html#ref-nps_hwa)).
Learn more about Eastern Hemlocks in [Godman and Lancaster](https://littlekidsbigadventures.com/abrams-creek.html#ref-godman2021hemlock) ([n.d.](https://littlekidsbigadventures.com/abrams-creek.html#ref-godman2021hemlock)).
Suggested rewordings: 
Walking on from the campground, you will enter a densely wooded area covered by the branches and needles of tall pine trees. You’ll encounter a few hills along the trail, but there’s never sustained climbing that will be needed on this section of the trail.
After crossing Kingfisher Creek for the first time, another crossing of this same creek awaits around 100 yards further from the first crossing. This one is sometimes a little bit easier than the first crossing.
The trail continues to Backcountry Site #1 at 1.3 miles from the trail head. This is a fairly frequently-used backcountry campsite, one used by those backpacking and camping overnight. The national park only allows camping at designated backcountry sites, or sites at frontcountry campgrounds–like the one you passed at the beginning of the hike.
The ‘#1’ refers to this site’s number among all of the backcountry sites in the park: there are around 70, total and this one happens to be numbered as the first! The number changes as sites are added, removed, or moved to ensure that the areas around the sites are not impacted too negatively by their frequent use - Hot take: I don’t know that I’d include all this information about the backcountry campsite. If this is for families who don’t hike much and are just getting started, this information doesn’t feel useful.
This is another great place to stop and rest. If you so choose, you can continue from here. Note the trail sign that is immediately before the backcountry site; it marks the intersection between the Cooper Road Trail, which you have been on, and the Little Bottoms Trail, which branches from the Cooper Road Trail here. To continue further, take the Little Bottoms Trail. You’ll soon cross a small creek and begin to climb—the steepest climb of this entire trail, and a quite steep climb by any metric. At the top of the climb atop this ridge, you’ll find another good spot to stop to rest at 1.5 miles.
At 2.0 miles, you’ll return to Abrams Creek for the first time since walking alongside it along the road near the campground! This is - again - a great place to stop, rest, and relax—and to splash in Abrams Creek or the small creek, Buckshank Branch, that feeds into it.
Information on bottomlands I could find. I can’t find anything substantive:
https://www.hikingproject.com/trail/7008278/little-bottoms-trail
Not related to bottomlands, but: “Before the first white settler arrived on the scene, Cades Cove had been a part of the Cherokee Indians' domain. Abrams Creek and Abrams Falls are features named for a prominent Cherokee chief named "Old Abram" who at one time lived in a village at Chilhowee on the Little Tennessee River. According to tradition, Old Abram's wife was named Kate, and Cades (Kate's) Cove was named after her.” - http://www.yoursmokies.com/cades-cove-history.html
http://smokyscout.blogspot.com/2010/08/so-good-to-be-back-again.html?m=1
21.5
I wouldn’t frame this as a possible “true adventure” for a new-to-hiking family. Maybe reword this as simply something nearby. Maybe just note the falls and provide access to a more accessible way to access it.
A true adventure—one that could take all day—is to continue on the Little Bottoms Trail past Backcountry Site #17. Doing so leads further (2.5 miles, one-way) to one of the most-loved waterfalls in the park, Abrams Falls. Note that the total distance of the hike from where you started (at the Abrams Creek Trailhead) to this waterfall would be around 11 miles, with some notable ascents and descents along the way. See the Abrams Falls trail entry in this book for another, shorter route to the Abrams Falls waterfall, if interested.
